sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 12.1.2 (Unreleased)
+## 12.1.2 (2020.05)
```

Comment:
The package.json shows `12.2.0` to be the version for the next update

Reply:
Based on the changelog, the changes are mainly bug fixes. Any idea why the minor version got updated in the package.json file?

Reply:
I think it was because of the move to core-http v1.1.0, from 1.0.3.

Reply:
`core-http` justified the minor version update due to the addition of the feature where we support the disabling of the response decompression.

For storage, this is not a feature, but a bug fix. So, I would say that a patch update should be sufficient.

Reply:
The main reason for core http minor version bump is the WebResourceLike change.  We thought it should be compatible but still bumped to be safe. I remember doing so for storage for the same rwason.  However, if it is not breaking anything I can revert back to the old minor versions.

Reply:
Well, if something is breaking then that would require a major version update. But regardless, I don't believe there is any net difference is usage for the end user of the storage library. So, I don't believe a minor version update is needed here

Reply:
Breaking not in "api breaking changes" but potential issues in the file-datalake/blob/core-http dependency chain and mix-match.  I reverted the storage version bump.  Hopefully we will get builds passing.